homeassistant/components/light/tcpbulbs.py

```diff
@@ -0,0 +1,146 @@
+"""

```

Comment:
As a side-note, would it make sense to rename this from tcpbulbs to be more specific, to avoid confusing users?

Reply:
Well the name of the product is TCP Connected, I added bulbs so they wouldn't think it's general lighting